ci/cloudbuild/README.md

```diff
@@ -1,23 +1,51 @@
-This directory contains the files for starting and configuring Google Cloud
-Build. The `cloudbuild.yaml` file is the main config file for cloud build.
-Cloud builds may be managed from the command line with the `gcloud builds`
-command. To make this process easier, the `build.sh` script can be invoked by
-users to trigger the builds specified in the `builds/` subdirectory. The
-`build.sh` script can run these builds on the user's local machine, or it can
-submit the builds to Cloud Build if a distro is specified with the
-`-d|--distro` flag. Distros are defined by the existence of a
-`Dockerfile.<distro>` file.
-
-## Examples
-
-The following command runs the `asan` build on the user's local machine.
-```
-$ ci/cloudbuild/build.sh asan
-$ ci/cloudbuild/build.sh -d local asan  # equivalent to above
-```
-
-The following command runs the `cmake+clang` build on the in Cloud Build using
-the `Dockerfile.fedora` image.
-```
-$ ci/cloudbuild/build.sh -d fedora cmake+clang
-```
+# Google Cloud Build
+
+This directory contains the files needed for our GCB presubmit ("PR builds")
+and postsubmit ("CI builds") builds. The `cloudbuild.yaml` file is the main
+config file for cloud build. Cloud builds may be managed from the command line
+with the `gcloud builds` command. To make this process easier, the `build.sh`
+script can be used to submit builds to GCB, run builds locally in a docker
+container, or even run builds directly in your local environment. See `build.sh
+--help` for more details.
+
+GCB triggers can be configured in the http://console.cloud.google.com/ UI, but
+we prefer to configure them with SCM controlled YAML files that live in the
+`triggers/` directory. Managing triggers can be done with the `gcloud` command
+also, but we have the local `trigger.sh` script to make this process a bit
+easier. See `trigger.sh --help` for more details.
+
+## Testing principles
+
+We want our code to work for our customers. We don't their exact environment
+and configuration, so we need to test our code in a variety of different
+environments and configurations. The main dimensions that we need to test are:
+
+- OS Platform: Windows, macOS, Linux x `N` different distros
+- Compilers: clang, gcc, msvc
```

Comment:
I was told those are supposed to be spelled Clang, GCC and MSVC.

Reply:
Fixed there and elsewhere.